tools/build_script_generator/scons/resources/SConstruct.in

```diff
@@ -72,6 +72,7 @@ program = env.Program(target=project_name+".elf", source=sources)
 
 # SCons functions
 env.Alias("cmakewrapper", env.CMakeWrapper())
+env.Alias("qtcreator", env.QtCreatorProject(sources))
```

Comment:
@rleh @chris-durand wasn't this supposed to include all files (ie. sources *and* headers)?

Reply:
Yes

Reply:
I remember this was divulged into some epic mess where the SConstruct now also had to track all the headers as well. Le Sigh™.

Reply:
Hm, I could just search all the include paths for the headers tho… 🤔

Reply:
Yes, the headers are missing. I'll fix that, rebase the branch and add the defines.

Reply:
No, I'm already working in it, modm/*sources are also missing.

Reply:
Sorry, I didn't refresh the page before I posted the comment and missed all above.

Reply:
No worries. I've now added searching the include paths for header files and explicitly the `./modm` path for sources. I've also made the paths relative, instead of absolute, so that they can actually be committed and used. This works fine for me with Qt Creator 4.7, can you check on your ends?

Reply:
> modm/*sources are also missing.

The xpcc generator didn't include the xpcc sources, only the project files and the board header from xpcc. I'm not sure if I would add the modm sources as they normally should not by edited by library users.

Reply:
Hm, ok. I've made this simply an option now, which defaults to off.

Reply:
> This works fine for me with Qt Creator 4.7, can you check on your ends?

It's working for me (Qt Creator 4.7).